cilium/cmd/config.go

```diff
@@ -73,15 +75,42 @@ func configDaemon(cmd *cobra.Command, opts []string) {
 			if err := command.PrintOutput(cfgStatus.Realized.Options); err != nil {
 				os.Exit(1)
 			}
+			if err := command.PrintOutput(cfgStatus.DaemonConfigurationMap); err != nil {
+				os.Exit(1)
+			}
 			return
 		}
+		fmt.Print("### Read-Only Configurations ###\n\n")
+		keys := make([]string, 0, len(cfgStatus.DaemonConfigurationMap))
+		for k := range cfgStatus.DaemonConfigurationMap {
+			keys = append(keys, k)
+		}
+		sort.Strings(keys)
+		for _, k := range keys {
+			v := cfgStatus.DaemonConfigurationMap[k]
+			if reflect.ValueOf(v).Kind() == reflect.Map {
+				mapString := make(map[string]string)
+				m, ok := v.(map[string]interface{})
+				if ok {
+					for key, value := range m {
+						mapString[key] = fmt.Sprintf("%v", value)
+					}
+					dumpConfig(mapString)
+					continue
+				} else {
+					fmt.Print("Error: cannot cast daemon config map to map[string]interface{}\n")
```

Comment:
To make sure that errors are separated from the output from the tool, please consider printing errors to stderr like this:

```suggestion
					fmt.Fprint(os.Stderr, "Error: cannot cast daemon config map to map[string]interface{}\n")
```

This can be helpful to ensure that the tool plays well with other unix tools (`grep`, etc.).